plugins/depthCamera/include/yarp/dev/DepthCameraDriver.h

```diff
@@ -267,9 +267,10 @@ class yarp::dev::GazeboYarpDepthCameraDriver:   public yarp::dev::DeviceDriver,
     unsigned char  *m_RGBPointCloud_Buffer;
     int counter;
 
+#if GAZEBO_MAJOR_VERSION < 7
     gazebo::sensors::CameraSensor*       m_imageCameraSensorPtr;
     gazebo::rendering::CameraPtr         m_imageCameraPtr;
```

Comment:
I see, I was missing this line : https://github.com/robotology/gazebo-yarp-plugins/blob/fix_gazebo7/plugins/depthCamera/src/DepthCameraDriver.cpp#L190 .